### DISCUSSION TITLE
Please Help with the new camera measure feature

### DISCUSSION THREAD

**Discussion by fnordson** [streetcomplete/StreetComplete#4000] [Q&A] [answered]
I tried it several times and I fail to grasp the functionality.  
There is an arrow on the road.  
I don't know what to do with it. I went there and it jump somewhere else. Went there, same thing.  
How is this supposed to work? 

I also can't imagine I'm the only one struggling here. I think some short video linked within the app (maybe on first use) could help here.

**Comment by 10992-osm** [comment, marked as answer]
By way of an immediate answer to your questions:

1. When the arrow appears, and is at the edge of the road, tap the screen,
2. The distance will appear in a white box, and an orange line will stretch out from the arrow,
3. When you've finished measuring, tap the screen again, and then either press the tick (✓) to confirm, or the undo button.

See #4001

**Reply by fnordson** [reply]
Aaah thank you very much! 
I'll try that tomorrow :)

**Reply by mnalis** [reply]
thanks, added to [FAQ](https://wiki.openstreetmap.org/wiki/StreetComplete/FAQ#How_does_one_use_AR_measure_feature?)
